include/util/dist_table_wrapper.hpp

```diff
@@ -46,6 +46,30 @@ template <typename T> class DistTableWrapper
         return table_[index];
     }
 
+    void InvalidateRoute(NodeID from, NodeID to)
```

Comment:
The DistTableWrapper is a wrapper around a vector that holds <T>, in the trip case the <T> is an EdgeWeight. This method here simply accepts NodeIDs to access the correct position of the necessary EdgeWeight in the table vector of EdgeWeights. So keeping the function as-is should be fine in this case.